### PR TITLE
[Feature] - Added Images component in the library

### PR DESCRIPTION
**Pull request by parthpandyappp**
### Directory structure and file justification
Added a new directory named alerts under which markup has been written in `images.html` and the stylesheet file is written in `images.css`. The stylesheet file `images.css` only consists of the styles related to buttons i.e it is a feature-specific stylesheet file that is later getting integrated with the main stylesheet file in the root directory i.e `styles.css`

### Explicit integration of stylesheet file
As we see, many open-source CSS libraries generally have a CDN link that is embedded and linked to the head section of any markup file. Similarly, a single main styles.css file is responsible to provide access to all the other stylesheets.

## About this PR
This PR adds an Images component of different types.

### CODE REVIEW

File: images/images.css [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  height: 16rem;
+  width: 16rem;
+}
+
+@media screen and (max-width: 750px) {
+  .disp-img {
+    padding: 1.5rem;
+  }
+  .img-responsive {
+    width: 250px;
```
Comment (yashpatel1797) [inline images/images.css]: You can use rem here as well.

**Review by yashpatel1797 — COMMENTED** [review]
Code looks good to me.

File: images/images.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (sonishreyas) [inline images/images.html:8]: you can add a author meta tag here

**Review by sonishreyas — APPROVED** [review]
Just left one suggestion. Rest looks goos to me

**Review by yashpatel1797 — APPROVED** [review]

File: images/images.css (hunk 1 above)
Comment (parthpandyappp) [reply images/images.css]: Yes man! Always miss out on converting pixel values to rem. Thanks for pointing it out.

**Comment by parthpandyappp** [comment]
Yup! Looked into it. Will update it in some time.

**Comment by parthpandyappp** [comment]
Fixed!

**Pull request merged**